src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -843,7 +844,9 @@
                 "jqueryui": location.pathname.replace(/\/[^/]+$/, "") + "js/lib/jquery-ui.min",
                 "chosen": location.pathname.replace(/\/[^/]+$/, "") + "js/lib/chosen.jquery.min",
                 "pips": location.pathname.replace(/\/[^/]+$/, "") + "js/lib/jquery-ui-slider-pips.min",
-                "underscore": location.pathname.replace(/\/[^/]+$/, "") + "js/lib/underscore-min"
+                "underscore": location.pathname.replace(/\/[^/]+$/, "") + "js/lib/underscore",
+                "underscore/underscore-min": location.pathname.replace(/\/[^/]+$/, "") + "js/lib/underscore",
+                "underscore/underscore": location.pathname.replace(/\/[^/]+$/, "") + "js/lib/underscore",
```

Comment:
I think this is capturing cases where underscore is brought in via a CDN. I'm attempting to redirect them to the framework underscore.